### DISCUSSION TITLE
Installing Klipper for the first time

### DISCUSSION THREAD

**Discussion by ericsch333** [bassamanator/Sovol-SV06-firmware#53] [General]
I ordered my SV06 plus and want to run Klipper I have a pi and screen and have been using octo print for a while which is the best way to install it. i have seen octo print , mainsail or just build klipper with klipper screen any help would be appreciated. thanks

**Comment by bassamanator** [comment]
Don't know anything about klipperscreen.

Just read all the documentation on the repo and follow the instructions precisely, if you want to get klipper running on your printer.

**Comment by bassamanator** [comment]
Just to clarify things if you weren't aware. 

Mainsail/Fluidd are like Octoprint. Meaning, if you use either, you don't need Octoprint. You could go Klipper + Octoprint instead, but that's resource intensive and unnecessary unless you have a special need for it.

So, I recommend using Mainsail or Fluidd. And then of course Klipper on the board.

**Reply by ericsch333** [reply]
Thanks I have i will do that
